packages/backend/src/scheduler.ts

```diff
@@ -1,3 +1,6 @@
+// organize-imports-ignore
+import './otel'; // must be imported first
```

Comment:
I'm concerned that importing this is going to interfere with the SIGINT handler that graphile provides. Usually it will gracefully shutdown queues, but otel.ts has it's own handler